packages/react-core/src/components/Modal/examples/Modal.md

```diff
@@ -13,84 +13,114 @@ import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
+
 ```ts file="./ModalTopAligned.tsx"
 ```
 
-### Small
+### Small modal
+
+To shorten the horizontal width of a modal, the `variant` property can be set to "small".
 
 ```ts file="./ModalSmall.tsx"
 ```
 
-### Medium
+### Medium modal
+
+To shorten the horizontal width of a modal, the `variant` property can be set to "medium".
 
 ```ts file="./ModalMedium.tsx"
 ```
 
-### Large
+### Large modal
+
+To shorten the horizontal width of a modal, the `variant` property can be set to "large".
 
 ```ts file="./ModalLarge.tsx"
 ```
 
 ### Custom width
 
+To choose a specific width for a modal, use the `width` property. This example has a `width` of "50%".
+
 ```ts file="./ModalCustomWidth.tsx"
 ```
 
 ### Custom header and footer
 
+To add a custom header and footer to a modal, set the `header` and `footer` properties to a custom implementation. The following example passes title components into both the header and the footer and also passes an icon to the footer.
+
 ```ts file="./ModalCustomHeaderFooter.tsx"
 ```
 
 ### No header or footer
 
+To exclusively present information in a modal, remove the `header` and/or `footer`.
+
 ```ts file="./ModalNoHeaderFooter.tsx"
 ```
 
 ### Title icon
 
+To add an icon before a modal’s title, use the `titleIconVariant`, which can be set equal one of the predefined variants -- "success", "danger", "warning", "info", and "default" -- or to an imported custom icon. The following example uses a "warning" variant.
+
 ```ts file="./ModalTitleIcon.tsx"
 ```
 
 ### Custom title icon
 
+To add a custom icon before a modal’s title, set `titleIconVariant` to an imported custom icon. The following example uses imports and uses a bullhorn icon.
+
 ```ts file="./ModalCustomTitleIcon.tsx"
 ```
 
 ### With wizard
 
+To guide users through a series of steps in a modal, you can add a [wizard](/components/wizard) to a modal. To configure the `<Wizard>`, pass an array that contains a “name” and “component” value for each step into the `steps` property.
+
 ```ts file="./ModalWithWizard.tsx"
 ```
 
 ### With dropdown
 
+To present a menu of actions or links to a user you can add a [dropdown](/components/dropdown) to a modal. To allow the dropdown to visually break out of the modal container, set the `menuAppendTo` property to “parent”. Handle the modal’s closing behavior by listening to the `onEscapePress` callback on the `<Modal>` component. This allows the `esc` key to collapse the dropdown without closing the entire modal.
+
 ```ts file="./ModalWithDropdown.tsx"
 ```
 
 ### With help
 
+To help simplify and explain complex models, add a help [popover](components/popover). Only place a help icon at the modal level if its information applies to all content in the modal. If the help popover is specific to a particular modal section, place the help icon beside that section instead.
```

Comment:
```suggestion
To help simplify and explain complex models, add a help [popover](/components/popover). Only place a help icon at the modal level if its information applies to all content in the modal. If the help popover is specific to a particular modal section, place the help icon beside that section instead.
```

Reply:
I will always manage to mess up links somehow 😆